libs/gpg45-evaluator/src/test/java/uk/gov/di/ipv/core/library/gpg45/domain/CredentialEvidenceItemTest.java

```diff
@@ -0,0 +1,402 @@
+package uk.gov.di.ipv.core.library.gpg45.domain;
+
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.MethodSource;
+import uk.gov.di.ipv.core.library.gpg45.Gpg45Scores;
+import uk.gov.di.ipv.core.library.gpg45.domain.CredentialEvidenceItem.EvidenceType;
+import uk.gov.di.ipv.core.library.gpg45.exception.UnknownEvidenceTypeException;
+
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+import java.util.stream.Stream;
+
+import static org.junit.jupiter.api.Assertions.*;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+class CredentialEvidenceItemTest {
+    @Test
+    void shouldConstructorWithEvidenceTypeAndScore() {
+        CredentialEvidenceItem credentialEvidenceItem1 =
```

Comment:
I think, you can create a method or can use ParameterizedTest here. There are so many duplicate code here.